tests/test_tinydb_provider.py

```diff
@@ -0,0 +1,191 @@
+# =================================================================
+#
+# Authors: Tom Kralidis <tomkralidis@gmail.com>
+#
+# Copyright (c) 2023 Tom Kralidis
```

Comment:
2024?

Reply:
Good catch!  Updated here and in `pygeoapi/plugin.py`.